tasks/release.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+tasks:
+  snapshot:
+    desc: Creates a snapshot of the current working directory with files in .drainpipeignore excluded. The target directory should be provided as a cli argument e.g. task release:snapshot -- /tmp/release
+    cmds:
+      - if [ ! -f ".drainpipeignore" ]; then echo ".drainpipeignore does not exist" && exit 1; fi
+      - if [ -d "{{.CLI_ARGS}}" ] || [ "" == "{{.CLI_ARGS}}" ]; then echo "Please provide a path to a directory that does not yet exist" && exit 1; fi
+      - cp -R . {{.CLI_ARGS}}
```

Comment:
Something to note is that this means that on "live" locals, files are copied to be removed. For example, I have files in `sites/default/files/...`, which total several GB. There's a bit of a penalty copying those to the other directory.